tests/next_tests/integration_tests/multi_feature_tests/iterator_tests/test_with_toy_connectivity.py

```diff
@@ -408,8 +408,12 @@ def shift_sparse_stencil2(inp):
     return list_get(1, list_get(3, neighbors(V2E, inp)))
 
 
-def test_shift_sparse_input_field2(program_processor_no_gtfn_exec, lift_mode):
-    program_processor, validate = program_processor_no_gtfn_exec
+def test_shift_sparse_input_field2(program_processor, lift_mode):
+    program_processor, validate = program_processor
+    if program_processor == gtfn_cpu.run_gtfn or program_processor == gtfn_cpu.run_gtfn_imperative:
+        pytest.xfail(
```

Comment:
Just to be sure: Your conclusion is that this test is failing for reasons unrelated to this PR?

Reply:
yes